rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import typescript from '@rollup/plugin-typescript';
+import dts from 'rollup-plugin-dts';
+
+const config = [
+    {
+        input: 'lib/index.js',
+        output: {
+            file: 'passwordless-api.js',
```

Comment:
add `dist`folder to make ignoring easier. Also change the `main:`-path to the dist.